### ISSUE TITLE
Deserialize null or missing fields as None?

### ISSUE THREAD

**Issue by BrianVallelunga** [Tarmil/FSharp.SystemTextJson#176]
I'm having a hard time figuring out exactly how to handle null or missing JSON fields when deserializing a record. My serializer settings are currently:

``` f#
let serializerOptions =
    JsonFSharpOptions
        .FSharpLuLike()
        .WithAllowNullFields(true)
        .WithSkippableOptionFields(true)
        .ToJsonSerializerOptions()
```

I don't care much about how an `Option` type gets serialized. Either `null` or as a missing field is fine by me, but when deserializing I want both `null` and missing fields to deserialize as `None`. Right now I'm getting errors if `null` is present. Is there a magic set of options that gets me the behavior I want? Thanks for the help.

**Comment by drhumlen** [comment]
We're having the exact same problem. It would be nice if it treated null and absence the same way when deserializing (with the given options).

**Comment by pkese** [comment]
Same here.

**Comment by martin-bmc** [comment]
I have the same issue. In an API I want to allow a user to either not send an optional field or send it as null,
However using the default options makes optional fields required and using `WithSkippableOptionFields `will set a null value to `Some null`. I would a configuration where they both would be None.

Also it seems like issue is similar to #151

**Comment by Tarmil** [comment]
This is released in v1.3.13:

```fsharp
JsonFSharpOptions.Default()
    .WithSkippableOptionFields(SkippableOptionFields.Always, deserializeNullAsNone = true)
```

**Comment by BrianVallelunga** [comment]
Thanks! I'm going to test this today.
